engine/table/src/main/java/io/deephaven/engine/table/impl/partitioned/PartitionedTableProxyImpl.java

```diff
@@ -613,8 +613,9 @@ public PartitionedTable.Proxy aggBy(Collection<? extends Aggregation> aggregatio
     @Override
     public PartitionedTable.Proxy updateBy(UpdateByControl control, Collection<? extends UpdateByOperation> operations,
             Collection<? extends ColumnName> byColumns) {
+        final Table emptyTarget = target.merge();
```

Comment:
My hope here is that `merge()` defers a lot of work until the column sources are queried and I can use this as a source or "original" column sources for reinterpretation/restoration. The operators do maintain a reference to this table's column sources which could conceivably be a problem.

If we need to release these, we could spawn a dummy, empty table with the same table-definition that would work as well for interpretation purposes.